package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "snapfu",
+	"name": "@searchspring/snapfu",
```

Comment:
The name is just 'snapfu'. It is already published:
https://www.npmjs.com/package/snapfu